Formula/yarn.rb

```diff
@@ -1,26 +1,19 @@
-require "language/node"
-
 class Yarn < Formula
   desc "JavaScript package manager"
   homepage "https://yarnpkg.com/"
   url "https://yarnpkg.com/downloads/0.21.3/yarn-v0.21.3.tar.gz"
```

Comment:
Plot foiled! In that case it would only help if yarnpkg.com went down.

I guess we could use `http://distfiles.gentoo.org/distfiles/yarn-v0.21.3.tar.gz` or be mirror-less.